src/go.mod

```diff
@@ -3,17 +3,22 @@ module github.com/containers/toolbox
 go 1.13
 
 require (
-	github.com/HarryMichal/go-version v1.0.0
+	github.com/HarryMichal/go-version v1.0.1
 	github.com/acobaugh/osrelease v0.0.0-20181218015638-a93a0a55a249
-	github.com/briandowns/spinner v1.10.0
+	github.com/briandowns/spinner v1.16.0
```

Comment:
Shouldn't this be updated to `v1.17.0` to include your fix for https://github.com/briandowns/spinner/issues/123 ?

Reply:
Sure. The change has been made before the release.